dipy/tracking/utils.py

```diff
@@ -956,3 +958,124 @@ def reduce_rois(rois, include):
             exclude_roi |= rois[i]
 
     return include_roi, exclude_roi
+
+def flexi_tvis_affine(sl_vox_order, grid_affine, dim, voxel_size):
+    """ Computes the mapping from voxel indices to streamline points,
+        reconciling streamlines and grids with different voxel orders
```

Comment:
What do you mean? I think the def is against the left edge, and the docstring is indented like the rest of the functions in this file?